src/components/InputField/InputField.styled.js

```diff
@@ -59,3 +59,23 @@ export const PasswordReapetLable = styled.p`
   padding-left: 14px;
   color: #49454f;
 `;
+
+export const EyeIcon = styled.div`
+  position: absolute;
+  right: 14px;
+  top: 16px;
+  cursor: pointer;
+
+  svg {
+    width: 24px;
+    height: 24px;
+    stroke: var(--color-grey-9);
```

Comment:
You don't need to hardcode a color for the stroke element itself for icons from the react icons lib. You just need to put a color property. Try to use color: var(--color-grey-9). Same for :hover selector.

Reply:
Done.